src/services/ChartService.js

```diff
@@ -243,9 +243,75 @@ const getColor = (index, isHover = false) => {
   return isHover ? hoverColors[index-1] : colors[index-1];
 };
 
+function FinancingSavingsBar (props){
+  // const monthlyPayments = 386.27;
+  // const monthlySavings = 458.81;
```

Comment:
What are these commented numbers?

Reply:
The values that were in the excel sheet for testing

Reply:
Should this be indicated somewhere? Or these values removed?

Reply:
Hi, the monthlypayments and monthlysavings are useful indications of affordability.